src/game/game.jsx

```diff
@@ -0,0 +1,304 @@
+import React from "react";
+import { Board } from "./board/board.jsx";
+import { Header } from "./header/header.jsx";
+import { Controls } from "./controls/controls.jsx";
+import { Menu } from "./menu/menu.jsx";
+import { Score } from './score/score.jsx';
+import s from "./game.module.css";
+
+export class Game extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = getFromLocalStorage('state') ?
+			this.state = getFromLocalStorage('state') :
```

Comment:
do this check in `ComponentDidMount`

Reply:
Thx for the good advice! Proposed solution:

```suggestion
export class Game extends React.Component {
	constructor(props) {
		super(props);
   	 	this.state = {
        		history: [
        		{
        		squares: Array(9).fill(null)
        		}
        		],
        		stepNumber: 0,
        		xIsNext: true,
        		single: false,
        		isMenuOpen: true,
        		score: [0, 0],
    		}
		this.cleanHistory = this.cleanHistory.bind(this);
		this.jumpTo = this.jumpTo.bind(this);
		this.changeMode = this.changeMode.bind(this);
		this.toggleMenu = this.toggleMenu.bind(this);
		this.changeScore = this.changeScore.bind(this);
		this.resetScore = this.resetScore.bind(this);
	}

  	componentDidMount() {
    		let stateObj = getFromLocalStorage('state');
    		if (stateObj) {this.setState(stateObj)};
  	}
```

Reply:
```
if (stateObj) {
   this.setState(stateObj)
};
```

Reply:
Okay!)

```suggestion
  componentDidMount() {
    let stateObj = getFromLocalStorage('state');
    if (stateObj) {
      this.setState(stateObj)
    };
  }
```